src/tests/unit/common/is-function.ts

```diff
@@ -0,0 +1,6 @@
+import { isFunction as checkIsFunction } from 'lodash';
+import { It } from 'typemoq';
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Copyright header should go above imports

Reply:
Ups